docs/manage-your-account/creating-environments.mdx

```diff
@@ -174,10 +174,10 @@ gitGraph
 ```
 
 In this example, we've added an `Admin` Role to the Production environment after copying it into the Development environment.
-When we merge the Development environment back into the Production environment, we have a conflict with the `Admin` Role.
+When we merge the development environment back into the production environment, we will have a conflict with the `Admin` role.
 
 When merging environments, you can choose to resolve conflicts by either:
-* **Fail**: (default) Fail the merge, rollback the entire merge operation and keep the existing object on the target environment.
+* **Fail**: (default) If the merge fails, the entire merge operation will be rolled back, and the existing object will remain in the target environment.
```

Comment:
Incorrect terminology. In case of merge conflict - it causes it to fail and roll back.
@danielbass37